docs/source/tensorboard.rst

```diff
@@ -0,0 +1,48 @@
+torch.utils.tensorboard
+===================================
+
+.. note::
+    TensorBoard logging is used to visualize PyTorch models and metrics
+    during their training process. Scalars, images, histograms, graphs, and
+    embedding visualizations are supported for PyTorch models and tensors
+    as well as Caffe2 nets and blobs.
+
+    The SummaryWriter class is your main entry to log data for consumption
+    and visualization by TensorBoard.
+
+    A simple example is
+
+    ::
+        import torch
+        import torchvision
+        from torch.utils.tensorboard import SummaryWriter
+        from torchvision import datasets, transforms
+
+        writer = SummaryWriter()
+
+        transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.5,), (0.5,))])
+        trainset = datasets.MNIST('mnist_train', train=True, download=True, transform=transform)
+        trainloader = torch.utils.data.DataLoader(trainset, batch_size=64, shuffle=True)
+        model = torchvision.models.vgg16(False)
+        images, labels = next(iter(trainloader))
+
+        grid = torchvision.utils.make_grid(images)
+        writer.add_image('images', grid, 0)
+        writer.add_graph(model, images)
+        for n in range(100):
+            writer.add_scalar('scalar1', torch.rand(1)[0], n)
```

Comment:
Umm... when would you ever want to do this? (It's more helpful for users if the example does something they would actually want to do, as opposed to a synthetic example that just exists to exercise the API.)

Reply:
Removing.